src/commcare_cloud/commands/deploy/command.py

```diff
@@ -100,6 +101,9 @@ def run(self, args, unknown_args):
                 print(color_error("Skipping formplayer because commcare failed"))
             else:
                 rc = deploy_formplayer(environment, args)
+
+        if rc == 0:
+            publish_deploy_event("deploy_success", deploy_component, environment)
```

Comment:
The event will not be published if `commcare` deploy succeeds, but `formplayer` fails. Is that intentional?